src/backend/_scss/application.scss

```diff
@@ -659,10 +659,47 @@ a {
 
 .track-names {
   color: $black;
-  margin-top: 16%;
+  margin-top: 0;
+}
+
+.social-buttons {
```

Comment:
@geekyd Why there is another `.social-buttons` class defined on line 712. Please remove this one.